src/homepageExperience/components/steps/cli/InstallDependencies.tsx

```diff
@@ -0,0 +1,301 @@
+import CodeSnippet from 'src/shared/components/CodeSnippet'
+import {SafeBlankLink} from 'src/utils/SafeBlankLink'
+import {
+  ComponentColor,
+  Button,
+  ButtonGroup,
+  Orientation,
+  Table,
+  BorderType,
+} from '@influxdata/clockface'
+
+import React, {FC, useEffect, useState} from 'react'
+import {useDispatch} from 'react-redux'
+import {getBuckets} from 'src/buckets/actions/thunks'
+import {event} from 'src/cloud/utils/reporting'
+
+export const InstallDependencies: FC = () => {
+  const dispatch = useDispatch()
+
+  useEffect(() => {
+    dispatch(getBuckets())
+  }, [])
+
+  const headingWithMargin = {marginTop: '48px', marginBottom: '0px'}
```

Comment:
let's move this outside the function component, so like:

```ts
const headingWithMargin = {marginTop: '48px', marginBottom: '0px'}

export const InstallDependencies: FC = () => {
```

If it stays inside the component, it'll get re-created on each render and react won't be able to understand that it's the same object.